src/models/event.js

```diff
@@ -877,16 +878,24 @@ utils.extend(module.exports.MatrixEvent.prototype, {
     },
 
     /**
-     * Returns the status of the event, or the replacing event in case `makeReplace` has been called.
+     * Returns the status of any associated edit or redaction
+     * (not for reactions/annotations are their local echo doesn't affect the orignal event),
+     * or else the status of the event.
      *
      * @return {EventStatus}
      */
-    replacementOrOwnStatus() {
+    getAssociatedLocalEchoStatus() {
```

Comment:
Maybe just `getAssociatedStatus`? `LocalEcho` seems a bit redundant.

Reply:
fixed in #960